filament/src/materials/ssao/bilateralBlur.mat

```diff
@@ -4,7 +4,8 @@ material {
         {
             type : sampler2dArray,
             name : ssao,
-            precision: medium
+            precision: medium,
+            filterable: false
```

Comment:
I think there's also this: filament/src/materials/ssao/bilateralBlurBentNormals.mat